src/lib/rc/sbus.cpp

```diff
@@ -231,6 +286,32 @@ bool
 sbus_input(int sbus_fd, uint16_t *values, uint16_t *num_values, bool *sbus_failsafe, bool *sbus_frame_drop,
 	   uint16_t max_channels)
 {
+#if defined(__PX4_POSIX_OCPOC)
```

Comment:
@bkueng I know we've had issues before with normal rc input drivers due to some differences in our hardware. This particular part was put together by a colleague, specifically for our flight controller. I'll try reverting back to the original function here and get back to you soon.